### PR TITLE
All operators with 100% of coverage

### PR DESCRIPTION
**Pull request by sgomezg**
All operators with 100% of coverage

### CODE REVIEW

File: sdk/src/main/scala/com/stratio/sparkta/sdk/OperatorConstants.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+object OperatorConstants {
+
+  //FIXME: Do we really need this?
+  final val EmptyString = ""
+  final val SpaceSeparator = " "
+  final val One = 1
+  final val Zero = 0
+  final val UnderscoreSeparator = "_"
+  final val NullValue = "None"
```
Comment (danielcsant) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/OperatorConstants.scala]: One and NullValue constants are used just in one class. Please move this constants there

Comment (sgomezg) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/OperatorConstants.scala]: I think is better to keep it in one constant object, and test it (I'll add the test soon). What do you think?

Comment (danielcsant) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/OperatorConstants.scala]: I think that by default, If a constant is used just in one class it should be declared just in that class. If later on this constant is used by other classes we can set it in this OperatorConstants.

**Comment by sgomezg** [comment]
Changes done.

**Pull request merged**